src/test/java/net/pms/test/formats/FormatFactoryTest.java

```diff
@@ -165,6 +165,7 @@ public final void testFormatRetrieval() {
 		testSingleFormat("test.wmv", "MPG", Format.VIDEO);
 		testSingleFormat("test.wtv", "MPG", Format.VIDEO);
 		testSingleFormat("test.mpa", "MPGAudio", Format.AUDIO);
+		testSingleFormat("test.mp1", "MPGAudio", Format.AUDIO);
```

Comment:
@Nadahar You've a such way to say that that it turn my mood down.
My fault, i was near sure that you was too attached at that verbose, it's why i never said anything, but i thought that after some time, you'll not matter anymore.